docs/snippets/ie_common.cpp

```diff
@@ -50,6 +50,24 @@ int main() {
     }
     //! [ie:get_input_tensor]
 
+    //! [ie:set_callback]
+    auto restart_once = true;
+    infer_request.SetCompletionCallback<std::function<void(InferenceEngine::InferRequest, InferenceEngine::StatusCode)>>(
+        [&, restart_once](InferenceEngine::InferRequest request, InferenceEngine::StatusCode status) mutable {
+            if (status != InferenceEngine::OK) {
+                // Process error code
+            } else {
+                // Extract inference result
+                InferenceEngine::Blob::Ptr output_blob = request.GetBlob(outputs.begin()->first);
+                // Restart inference if needed
+                if (restart_once) {
+                    request.StartAsync();
+                    restart_once = false;
+                }
+            }
+        });
+    //! [ie:set_callback]
```

Comment:
but where is it inserted to the main documentation https://github.com/openvinotoolkit/openvino/blob/master/docs/OV_Runtime_UG/migration_ov_2_0/common_inference_pipeline.md#4-create-an-inference-request ? I suppose we should add several words and snippet to the `creation of infer request` section